File: PMS-API/PMS/PMS.Model/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PMS.Model
{
    public class Product
    {
        
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public string ProductCode { get; set; }
        public int Quantity { get; set; }
        public double Price { get; set; }
        public string Description { get; set; }
        public byte[]? image { get; set; }
        public int SubCategoryId { get; set; }
        public int CategoryId { get; set; }
    }
}

File: PMS/src/Models/Product.js
export default class Product{
    constructor(productName,productCode,quantity,price,description,CategoryId,SubCategoryId){
        this.productName = productName;
        this.productCode = productCode;
        this.quantity = quantity;
        this.price = price;
        this.description = description;
        this.CategoryId = CategoryId;
        this.SubCategoryId = SubCategoryId;
    }
}